ddoc/source/preprocessor.d

```diff
@@ -346,7 +346,13 @@ auto genChangelogVersion(string fileName, string fileText)
             }
             else
             {
-                macros ~="\n$(CHANGELOG_NAV %s, %s)".format(versions[el.index - 1], versions[el.index + 1]);
+                const prevVersion = versions[el.index - 1].to!string;
+                auto nextVersion = versions[el.index + 1].to!string;
+                // the next version is the beta release
+                if (el.index == versions.length - 2 && hasPrerelease)
+                    nextVersion = std.array.replace(nextVersion, "_pre", "");
```

Comment:
I split it up and didn't do it for all strings because in the future we might want to extend the CHANGELOG_NAV macro to allow for aliases, s.t. it could be e.g. "2.083 (Beta)" and I also think {prev,next}Version is clearer to the reader.